src/Foundatio/Caching/CacheClientHelpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Foundatio.Caching
```

Comment:
ICacheClient.cs just defines the interface.  I can put the class/extension in that file if you prefer.